src/Common/src/System/IO/PathInternal.cs

```diff
@@ -22,7 +22,7 @@ internal static void CheckInvalidPathChars(string path)
                 throw new ArgumentNullException(nameof(path));
 
             if (HasIllegalCharacters(path))
-                throw new ArgumentException(SR.Argument_InvalidPathChars, nameof(path));
+                throw new ArgumentException(SR.Format(SR.Argument_InvalidPathChars, path), nameof(path));
```

Comment:
`Argument_InvalidPathChars` message does not have argument reference, so the path won't show up in the message.

Reply:
Fixed.